### DISCUSSION TITLE
Level position

### DISCUSSION THREAD

**Discussion by tripsylon** [bangcorrupt/hacktribe#297] [Ideas]
Would it be possible to write a script where shift +level combination could take you to the tracks assigned volume position? 

Thank you

**Comment by bangcorrupt** [comment]
The parameter value saved in the pattern file?  This would be useful and probably not too hard to implement.  I thought there was an option to compare the current state and saved state like in older models, but I can't find it.

**Reply by tripsylon** [reply]
Yes the value saved in the pattern file.  In knob mode we have a catch function, so maybe if the shift+volume combo could temporarily activate it even when you are in catch mode .

**Comment by ptowns** [comment]
Left Right arrows scroll through peramiter settings at the bottom of the screen. Very small nut shows current value.
